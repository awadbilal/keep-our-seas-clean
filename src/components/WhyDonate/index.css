.titles {
  color: #ffa60c;
  display: inline;
  border-bottom: 0.2rem solid #ffa60c;
  font-weight: bold;
}
.main.pt-5.pb-5.fdb-block.container-fluid {
  background-color: #eaeaea;
}

a.donateButton.btn.btn-primary {
  color: #000000;
  background-color: #ffbe4b;
  border-color: #ffbe4b;
  border-radius: 1.5rem;
  padding: 0.7rem 1.7rem;
}

a.donateButton.btn.btn-primary:hover {
  color: #fff;
  background-color: #ff9f0a;
  border-color: #ff9f0a;
}
img.whyDonate {
  max-width: 100%;
}
