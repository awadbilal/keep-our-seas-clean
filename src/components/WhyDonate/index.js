import React from "react";
import { Row, Col, Container, Button } from "react-bootstrap";
import whyDonateImage from "../../images/whyDonate.png";
import { useTranslation } from "react-i18next";
import "./index.css";

const WhyDonate = () => {
  const { t } = useTranslation();

  return (
    <Container fluid className="main pt-5 pb-5 fdb-block">
      <Row className="align-items-center">
        <Col sm={12} md={{ span: 4, offset: 2 }}>
          <h2 className="titles">{t("whyDonate.title")}</h2>
          <p className="mt-4">{t("aboutUsContent.title.paragraph")}</p>
          <Button className="donateButton" href="donate">
            {t("donate.button")}
          </Button>
        </Col>
        <Col sm={12} md={6}>
          <img className="whyDonate" src={whyDonateImage} alt="whyDonate" />
        </Col>
      </Row>
    </Container>
  );
};

export default WhyDonate;
