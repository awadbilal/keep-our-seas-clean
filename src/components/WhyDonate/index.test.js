import React from "react";
import renderer from "react-test-renderer";
import WhyDonate from "./index";

it("renders correctly", () => {
  const tree = renderer.create(<WhyDonate />).toJSON();
  expect(tree).toMatchSnapshot();
});
