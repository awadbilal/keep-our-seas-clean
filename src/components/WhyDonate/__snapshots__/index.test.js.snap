// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="main pt-5 pb-5 fdb-block container-fluid"
>
  <div
    className="align-items-center row"
  >
    <div
      className="col-md-4 col-sm-12 offset-md-2"
    >
      <h2
        className="titles"
      >
        whyDonate.title
      </h2>
      <p
        className="mt-4"
      >
        aboutUsContent.title.paragraph
      </p>
      <a
        className="donateButton btn btn-primary"
        href="donate"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        donate.button
      </a>
    </div>
    <div
      className="col-md-6 col-sm-12"
    >
      <img
        alt="whyDonate"
        className="whyDonate"
        src="whyDonate.png"
      />
    </div>
  </div>
</div>
`;
